python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_isis.schema.yml

```diff
@@ -135,7 +135,7 @@ keys:
                   - md5
                   - sha
                   - text
-                  - shared_secret
+                  - shared-secret
```

Comment:
This is a behavioral change. I can see from the template that this never worked.

Reply:
since this never worked as documented we don't need a porting guide entry.